prism.js

```diff
@@ -779,6 +779,89 @@ if (Prism.languages.markup) {
 Prism.languages.js = Prism.languages.javascript;
 
 
+/* **********************************************
```

Comment:
Please don't change this file. (It should be restored anyway once you remove the changes in the gulpfile)
This is the file used by Prism's website. There is no need for Solidity there.